backend/molgenis-emx2-webapi/src/test/java/org.molgenis.emx2.web/WebApiSmokeTests.java

```diff
@@ -61,6 +61,10 @@ public static void before() throws Exception {
         // disable because of parallism issues .and(MOLGENIS_INCLUDE_CATALOGUE_DEMO, "true")
         .execute(() -> RunMolgenisEmx2.main(new String[] {}));
 
+    // wait until started, is this worth it or should we go back to simply
+    // 'MolgenisWebservice.start(port)'?
+    Thread.sleep(5000);
```

Comment:
this looks dodgy , a test that needs to wait for 5 seconds ..

Reply:
yes, this shouldn't be here.